Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -32,7 +32,7 @@ class SentryFeedback: NSObject, SentrySerializable {
         super.init()
     }
     
-    func serialize() -> [String: Any] {
+    public func serialize() -> [String: Any] {
```

Comment:
Curious why do you set this function public if the class is not public?

Reply:
oh, leftover from when i thought the class needed to be public, didn't remove this one again 👍🏻